pages/drivers/drivers-list.tsx

```diff
@@ -13,14 +14,24 @@ import { getDriverList } from "../../services/drivers-list/drivers-list.service"
 
 const driverslist: NextPage = (props: any) => {
     const { driverslistProps } = props;
+    const [filteredDrivers , setFilteredDrivers] = useState(driverslistProps);
+
+    const handleSearchInput = (inputValue ?: any ) => {
+
+        let filterList = driverslistProps.filter((driver ?: any) =>{
+            return driver.firstName.toLowerCase().includes(inputValue) || driver.lastName.toLowerCase().includes(inputValue);
+        })
+        setFilteredDrivers(filterList);
+    }
+
     return(
         <Stylewrapper className='drivers-list'>
-            <Header {...config?.header} enableSearchBar={true}/>
+            <Header {...config?.header} enableSearchBar={true} onInputChange={handleSearchInput}/>
             <div className='drivers-list-profile'>
-                {driverslistProps.map((driver : any, key: any) => (
+                {filteredDrivers.map((driver : any, key: any) => (
```

Comment:
Maybe you can have a "no result" component in case filteredDrivers.length is 0 ?